README.md

```diff
@@ -10,36 +9,101 @@ license](APACHE-2.0.txt) and documentation is provided under the [CC-BY-4.0
 license](CC-BY-4.0.txt). For details about this dual-license structure, please
 see [`LICENSE.txt`](LICENSE.txt).
 
-## Building ##
+## Usage ##
 
-The `build-images.sh` script will build the docker images for all the Dockerfiles in the versioned
-folder directories.
+**Start a JanusGraph server instance**
 
-## Publishing ##
+The following command starts a JanusGraph instance using the latest version. The default configuration
+uses the [Oracle Berkeley DB Java Edition][JG_BDB] storage backend and the [Apache Lucene][JG_LUCENE]
+indexing backend.
 
-The `push-images.sh` script will push the docker images for all the versioned folders in the repo.
+```bash
+$ docker run --name janusgraph-default janusgraph/janusgraph:latest
+```
 
-Prior to publishing, you'll need to login to [Docker Hub][DH] using the `docker login` command.
+The following command starts the same JanusGraph instance using [docker-compose.yml](docker-compose.yml)
+
+```bash
+$ docker-compose up
+```
+
+**Connect to JanusGraph using Gremlin shell**
+
+The following command starts another JanusGraph container and allows remote connection to the original
+JanusGraph server instance. Add --network option (e.g. `--network janusgraph-docker_default`) if the
+server container is in another network.
+
+```bash
+$ docker run --rm --link janusgraph-default:janusgraph -it janusgraph/janusgraph:latest ./bin/gremlin.sh
+
+         \,,,/
+         (o o)
+-----oOOo-(3)-oOOo-----
+plugin activated: janusgraph.imports
+plugin activated: tinkerpop.server
+plugin activated: tinkerpop.utilities
+plugin activated: tinkerpop.hadoop
+plugin activated: tinkerpop.spark
+plugin activated: tinkerpop.tinkergraph
+gremlin> :remote connect tinkerpop.server conf/remote.yaml
+==>Configured janusgraph/172.17.0.2:8182
+gremlin> :> graph.addVertex("name", "chris")
```

Comment:
Please don't use the graph API any more in examples. We should only use the graph traversal API (see also JanusGraph/janusgraph#1108). so, this should be:
```groovy
g.addV('person').property('name', 'chris')
```